open-company/open-company-web
# Fix post preview rounded corners

Card: https://trello.com/c/ouCyIo4l

To test:
- add a post with an image
- [x] do you see the image preview in the stream?
- [x] does it has rounded corner?
- [x] does it NOT has a play button over it?
- add a post with a YouTube video
- [x] do you see the video preview in the stream?
- [x] does it has rounded corner?
- [x] does it has a play button over it?
- add a post with a Vimeo video
- [x] do you see the video preview in the stream?
- [x] does it has rounded corner?
- [x] does it has a play button over it?
- add a post with a Loom video
- [x] do you see the video preview in the stream?
- [x] does it has rounded corner?
- [x] does it has a play button over it?
- add a post with a GIF
- [x] do you NOT see the GIF preview in the stream?
